### ISSUE TITLE
Feature Request: Font size increase decrease option for the commit list TableView

### ISSUE THREAD

**Issue by bravobohan** [ForkIssues/Tracker#445]
Fork is great!

An option in the preference to choose a font size for the commit list (NSTableView?) will be great for usage on different display sizes. 

Cheers.

**Comment by DanPristupov** [comment]
How do you solve that problem with other native applications like Finder, etc?

**Comment by aparajita** [comment]
@DanPristupov I use Path Finder, which lets me change the font size of the UI and lists.

**Comment by r-owen** [comment]
I agree that the text is a bit too small for comfort. The ability to make it larger would be most welcome.

**Comment by omichde** [comment]
Weighting in on the font size topic: IMHO too small for comfortable reading.
(and I killed two remote branches by accident because Remotes are too close in the left pane, wish there would be a separator or something like that) - but admittedly that was my fault.

**Comment by danwilliams** [comment]
> How do you solve that problem with other native applications like Finder, etc?

System settings -> Accessibility -> Display -> Text -> Text size

<img width="827" alt="image" src="https://github.com/fork-dev/Tracker/assets/365912/214bcd30-f601-4e2d-bd8e-c049e2331bcd">

Note that you can adjust Finder individually if required.

Note also that, from my understanding, this ability came in with Sonoma - but it is certainly the current answer to the question raised, as of 2024.

**Comment by SWAGnus-Carlsen** [comment]
@danwilliams lad, it just only changes in calendar, finder etc. But how is this related to Fork?

**Comment by danwilliams** [comment]
> @danwilliams lad, it just only changes in calendar, finder etc. But how is this related to Fork?

It was in reply to a specific question (which I quoted).

**Comment by Tensho** [comment]
Is Apple going to let text font size customisation for 3rd party apps by any chance?
